microsoft/testsuites/acc/bvt.py

```diff
@@ -0,0 +1,149 @@
+import time
+from typing import Optional
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import SGXEnabled
+from lisa.schema import NodeSpace
+from lisa.tools import Lscpu
+
+
+@TestSuiteMetadata(
+    area="ACC_BVT",
+    category="functional",
+    description="""
+    This Vasic Validation Test (BVT) suite validates the availability of Secure Guard Extensions (SGX) on a given
+    platform.
+    """,
+)
+class ACCBasicTest(TestSuite):
+    @TestCaseMetadata(
+        description="""
+
+        This case verifies if the VM is SGX Enabled.
+
+        Steps:
+            1. Add keys and tool chain from Intel-SGX, LLVM and Microsoft repositories.
+            2. Install DCAP driver if missing.
+            3. Install required package.
+            3. Run Helloworld and Remote Attestation tests.
+
+        """,
+        priority=1,
+    )
+
+    def verify_SGX(self, log: Logger, node: Node) -> None:
+
+        if self.node.os.information.version != "20.4.0":
+                raise UnsupportedDistroException(self.node.os)
```

Comment:
is it only support on ubuntu 20.04? should we check the os type? 
```
            if (
                isinstance(self.node.os, Ubuntu)
                and self.node.os.information.version != "20.4.0"
            ):
```

Reply:
UnsupportedDistroException will cause case fail, skipexception can be raised here.

Reply:
Yes, currently this case only supports one Ubuntu version. I changed it to 18.04 for ease. Sure, I will use skip Exception instead.